main.tf

```diff
@@ -78,6 +80,64 @@ resource "aws_cloudwatch_event_bus" "this" {
     }
   }
 
+  dynamic "log_config" {
+    for_each = var.logging != null ? [var.logging] : []
+    content {
+      include_detail = log_config.value.include_detail
+      level          = log_config.value.level
+    }
+  }
+
+  tags = var.tags
+}
+
+resource "aws_cloudwatch_log_delivery_source" "this" {
+  count = local.create_logging ? 1 : 0
+
+  region = var.region
+
+  name         = coalesce(var.log_delivery_source_name, var.bus_name)
+  log_type     = "${upper(var.logging.level)}_LOGS"
+  resource_arn = aws_cloudwatch_event_bus.this[0].arn
```

Comment:
this does not include the scenario where the event bus is created outside this module